### PR TITLE
Add fluent ui dependency in json file.

### PR DESCRIPTION
**Pull request by luism3861**
# Why
this PR add fluent ui dependency in json file.!

### CODE REVIEW

File: react-native-libraries.json
Comment (brentvatne) [inline react-native-libraries.json:6178]:
```suggestion
    "android": false,
```

Comment (luism3861) [reply react-native-libraries.json]: i change your suggestion thank you @brentvatne !.

File: react-native-libraries.json
Comment (brentvatne) [inline react-native-libraries.json:6180]:
```suggestion
    "expo": true,
```

Comment (brentvatne) [reply react-native-libraries.json:6180]: it looks like this doesn't have native dependencies

File: react-native-libraries.json
Comment (brentvatne) [inline react-native-libraries.json:6176]:
```suggestion
    "npmPkg": "@fluentui/react-native",
```

Comment (luism3861) [reply react-native-libraries.json]: thanks i change all your suggestions thank you @brentvatne !.

**Comment by brentvatne** [comment]
i spoke with a friend at microsoft and we're going to hold off on landing this for now, they want to work on it a bit more before publicizing it further

**Comment by brentvatne** [comment]
i'll leave this open for later actually

**Comment by luism3861** [comment]
> i'll leave this open for later actually



> i'll leave this open for later actually

thanks

> i'll leave this open for later actually



> i spoke with a friend at microsoft and we're going to hold off on landing this for now, they want to work on it a bit more before publicizing it further

this dependency looks really great!!. thanks @brentvatne .

File: react-native-libraries.json
Comment (FalseLobster) [inline react-native-libraries.json:6183]: As one of the maintainers of fluentui-react-native, I'm not sure what some of these fields mean, but naively I would expect macos to be true as well as android and maybe expo should be false?

Comment (Simek) [reply react-native-libraries.json:6183]: @FalseLobster Thank you for the response! 👍 

The description of the fields can be found in Readme - https://github.com/react-native-community/directory#library-fields-description - basically they mean if the package works with selected out of three platform (links are in Readme). 🙂 

Also this PR is quite old and needs rebase as well as the package should be moved at the end of the list, @luism3861 would you like to take care of that?

Comment (brentvatne) [reply react-native-libraries.json:6183]: @FalseLobster - does fluentui-react-native ship native code or does it just depend on react-native primitives? if it just depends on react-native primitives (or common libraries used in the ecosystem, like react-native-gesture-handler, reanimated, and [others included in Expo SDK](https://docs.expo.dev/versions/latest/)) then it will work fine in expo go and on https://snack.expo.io

**Review by asklar — APPROVED** [review]

**Comment by Simek** [comment]
Closing in favor of #696.

**Pull request closed without merging**
